Cargo.toml

```diff
@@ -21,8 +21,7 @@ rand = "0.4"
 clippy = {version = "*", optional = true}
 
 [dependencies.kvproto]
-path = "../kvproto"
-#git = "https://github.com/pingcap/kvproto.git"
+git = "https://github.com/pingcap/kvproto.git"
```

Comment:
Since this is a library without a `Cargo.lock` this is very dangerous. It'd be safer to lock it to a particular tag or revision.